easybuild/easyconfigs/r/Ruby/Ruby-2.3.1.eb

```diff
@@ -0,0 +1,63 @@
+#    With <3 for EasyBuild
+#
+#    EasyConfig for Ruby 2.1:
+#    ----------------------------------------------------------------------------
+#    Copyright: 2014 - Gregor Mendel Institute of Molecular Plant Biology GmBH
+#    License: MIT
+#    Authors: Aaron <aaron.zauner@gmi.oeaw.ac.at> Zauner
+#    ----------------------------------------------------------------------------
+
+name = 'Ruby'
+version = '2.3.1'
+
+homepage = 'https://www.ruby-lang.org'
+description = """Ruby is a dynamic, open source programming language with 
+ a focus on simplicity and productivity. It has an elegant syntax that is 
+ natural to read and easy to write."""
+
+toolchain = {'name': 'dummy', 'version': 'dummy'}
```

Comment:
Yes, but my goal was to obtain a `travis` module that I could use whenever, so regardless of which toolchain is loaded...